NuRadioReco/framework/event.py

```diff
@@ -128,7 +128,30 @@ def set_id(self, evt_id):
     def get_run_number(self):
         return self.__run_number
 
-    def get_station(self, station_id):
+    def get_station(self, station_id=None):
+        """
+        Returns the station for a given station id.
+
+        Paramters
+        ---------
+
+        station_id: int
+            Id of the station you want to get. If None and event has only one station
+            return it, otherwise raise error. (Default: None)
+
+        Return
```

Comment:
Returns

Reply:
Even if we return a single station :o ;)